### PR TITLE
Inline webpack sass-loader options

### PR DESCRIPTION
**Pull request by joshlory**
Cleanup for PR https://github.com/code-dot-org/code-dot-org/pull/19706.

### CODE REVIEW

**Comment by joshlory** [comment]
Actually I have a mistake here 😕.  I need to set the `includePaths` option on `sass-loader`, not the `include` setting.  The way of setting options [is cleaner in webpack 2](https://stackoverflow.com/a/41779562), but I think I can do this in a backward-compatible way.

Updating this PR...

File: apps/webpack.js [hunk 1]
```diff
@@ -39,15 +41,12 @@ var baseConfig = {
...
-  sassLoader: {
-    includePaths: [path.resolve(__dirname, '..', 'shared', 'css')]
-  },
   module: {
     loaders: [
       {test: /\.json$/, loader: 'json-loader'},
       {test: /\.ejs$/, loader: 'ejs-compiled-loader'},
       {test: /\.css$/, loader: 'style-loader!css-loader'},
-      {test: /\.scss$/, loader: 'style-loader!css-loader!sass-loader'},
+      {test: /\.scss$/, loader: `style-loader!css-loader!sass-loader?includePaths=${scssIncludePath}`},
```
Comment (islemaster) [inline apps/webpack.js:49]: This looks like we'll _only_ convert scss that's in /shared/css, not anywhere else.  Is that the desired behavior, or am I not reading this correctly?

Comment (joshlory) [reply apps/webpack.js:49]: I think it converts .scss files everywhere, but the `includePaths` hint tells `node-sass` where to look for resolving `@import` statements: https://github.com/sass/node-sass#includepaths.

This shouldn't change the behavior at all — just moves the option from top-level `sassLoader` config option to the loader rule.

Comment (islemaster) [reply apps/webpack.js:49]: Oh, of course! :man_facepalming: I should know that.  LGTM!

Comment (joshlory) [reply apps/webpack.js:49]: No worries, I didn't realize it was an option passed to `node-sass` and not webpack!

**Pull request merged**